### PR TITLE
Add e2e-aws-serial to cluster-api-provider-aws

### PR DESCRIPTION
**Pull request by stbenjam**
This change adds e2e-aws-serial to this repo.

### CODE REVIEW

**Comment by JoelSpeed** [comment]
/hold

Would like to see the result of the rehearsal before we merge

**Comment by JoelSpeed** [comment]
/hold cancel

They all passed, I'm happy for them to be added

**Review by elmiko — APPROVED** [review]
/lgtm

**Pull request merged**
